tests/storage/iso/conftest.py

```diff
@@ -26,10 +26,7 @@ def copy_tools_iso_to_iso_sr(host, sr, location=None):
     # copy the ISO file to the right location
     iso_path = host.ssh(['find', '/opt/xensource/packages/iso/', '-name', '"*.iso"'])
     iso_new_name = sr.uuid + "_test.iso"
-    if location is not None:
-        iso_new_path = f"{location}/{iso_new_name}"
-    else:
-        iso_new_path = f"/run/sr-mount/{sr.uuid}/{iso_new_name}"
+    iso_new_path = f'{location}/{iso_new_name}' if location is not None else f'/run/sr-mount/{sr.uuid}/{iso_new_name}'
```

Comment:
Mostly less readable because of the length of the paths. Not a blocker, but maybe that's not a rule that is really important to enforce.

![image](https://github.com/user-attachments/assets/65f2aa32-b62d-436a-98bf-6b9e755bd039)